src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -156,6 +156,11 @@ export class NetCoreTaskHelper implements TaskHelper {
 
         runOptions.volumes = await this.inferVolumes(context.folder, runOptions, helperOptions, ssl, userSecrets); // Volumes specifically are unioned with the user input (their input does not override except where the container path is the same)
 
+        // Use a command that keeps the container running when started with the '-t' option.
+        if (runOptions.os !== 'Windows') {
+            runOptions.command = runOptions.command || '/bin/sh';
+        }

```

Comment:
This is an additional change.

To test this, I had used an image that by default prints usage instructions. This caused the debug session to fail because it assumed the default command would keep the container running.

This explicitly specifies a default command so it has the desired behavior.

@bwateratmsft what do you think about this change?

Reply:
My gut feeling is that it should be fine. We do rely on that behavior (of the container staying up and idle); I can't see it being harmful to make it explicit instead of assuming the base images (or in this case, the start command) remain unchanged. Similarly, in Visual Studio, the containers get started with `tail -f /dev/null` as the do-nothing command.

@karolz-ms or @philliphoff can you think of any issues that might arise?

Reply:
Hmm. I am not sure about this change. It means that, without additional information, VS Code will not use default command specified by the Docker image. This violates principle of least surprise IMO. And if the image requires a nonstandard command to be "useful", well, we already have a `command` property for that on the `docker-run` task.

Reply:
The alternative way to make it explicit would be to scaffold it in, but I don't want to stuff the tasks with so much stuff that it gets overwhelming. I'd worry that scaffolding something like that would cause more problems/confusion than it would solve.